packages/styles/src/elements/list-check.scss

```diff
@@ -0,0 +1,45 @@
+@use '../functions/tokens';
+@use '../tokens/elements';
+@use '../mixins/icons' as icon-mixin;
+@use '../functions/icons';
+
+tokens.$default-map: (
+  post-list-check-sizing-icon: 32px,
+  post-list-check-text-padding: 1px,
+  post-list-check-icon-gap-inline: 16px,
+  post-list-check-item-gap-block-text: 16px,
+  post-list-check-margin-block: 8px,
+  post-list-check-color-icon-fg: white,
+  post-list-check-color-text-fg: #050400,
+  post-list-check-padding-icon: 4px,
+  post-list-check-line-height: 30px
+);
+
+.list-check {
+  display: grid;
+  grid-template-columns: [marker] auto [item] 1fr;
+  list-style: none;
+  gap: tokens.get('list-check-item-gap-block-text') 0;
+  padding: 0;
+
+  > li {
+    display: grid;
+    grid-template-columns: subgrid;
+    column-gap: tokens.get('list-check-icon-gap-inline');
+    grid-column: span 2;
+    padding: tokens.get('list-check-text-padding') 0;
+
+    &::before {
+      margin-top: calc(-1 * tokens.get('list-check-text-padding'));
+      margin-bottom: calc(-1 * tokens.get('list-check-text-padding'));
```

Comment:
Please use `inline` and `block` properties instead of `right`, `left`, `top`, and `bottom`.